lib/registration.pm

```diff
@@ -119,8 +119,12 @@ sub fill_in_registration_data {
         if (get_var('SCC_ADDONS')) {
             for my $addon (split(/,/, get_var('SCC_ADDONS', ''))) {
                 if (check_var('VIDEOMODE', 'text')) {
-                    send_key_until_needlematch "scc-module-$addon", 'tab';
-                    send_key "spc";
+                    send_key_until_needlematch 'scc-module-area-selected', 'tab';
+                    send_key_until_needlematch "scc-module-$addon",        'down';
+                    send_key 'spc';
+                    for (1 .. 15) {    # go at beginning of the module list
```

Comment:
magic numbers are evil. Are you sure this will also work for SP1/SP2/SP3?
Why not use `send_key_until_needlematch 'scc-module-list-begin', 'up';`?